backend/functions/cases/cases.function.ts

```diff
@@ -21,26 +22,33 @@ const httpTrigger: AzureFunction = async function (
   functionContext: Context,
   casesRequest: HttpRequest,
 ): Promise<void> {
+  // Setup dependencies
   const applicationContext = await applicationContextCreator(functionContext);
   const casesController = new CasesController(applicationContext);
-  let caseChapter = '';
+
+  // Process request message
+  const caseId = casesRequest?.params?.caseId;
```

Comment:
Is caseId now required?  By default, if none is provided, then caseId will be undefined.  Is that what we want?

Reply:
Looks like that would be fine.

Reply:
CaseId is not required. Logic will try to grab that value but if the caseId is not set it doesn't return the case details.